src/transformers/models/t5/modeling_t5.py

```diff
@@ -1309,6 +1314,13 @@ def forward(
                 raise ValueError("You have to initialize the model with valid token embeddings")
             inputs_embeds = self.embed_tokens(input_ids)
 
+            if self.project_scalars is not None:
```

Comment:
You should also check that encoder_input_scalars_indices and encoder_input_scalars_values are not None. 
For a case it was defined but not used by a specific task.